README.md

```diff
@@ -63,22 +63,116 @@ In this dialog, options should be filled out as follows:
+### Release 1.0.0.3
+
+* **Breaking change**: Navigation names used by the connector now include the \<Report Owner\> as part of the comma separated name to distinguish between versions of a report name owned by different users. Published reports use the name DAS_PUBLIC for the report owner. When first opening existing Power BI files with connections to mobie snapshots prior to this release, edit the query and include the report owner name in the comma separated name.
+* Security Associations table included as a mechanism to implement dynamic row level security in your Power BI report.
+* Support ability to connect to FDASPROPT test table.
+* Improve error checking of incoming options and make parameters optional that are not always required.
+* By default, users only see snapshots that they have published themselves to the mobie DataBox. To retain the past behavior of seeing all snapshots, it is necessary to create a table called rn__Options. See below for instructions.
+
+#### Instructions for creating rn__Options table
+
+For SQL Server, you may run the following script (substituting your DataBox database name for MyDataBoxDatabase:
+
+```
+   USE [MyDataBoxDatabase]
+   GO
+   SET ANSI_NULLS ON
+   GO
+   SET QUOTED_IDENTIFIER ON
+   GO
+   CREATE TABLE [dbo].[rn__Options](
+	   [Name] [nvarchar](32) NOT NULL,
+	   [ReportUniqueName] [nvarchar](275)  NOT NULL,
+	   [Value] [nvarchar](256) NOT NULL,
+    CONSTRAINT [PK_rn__Options] PRIMARY KEY CLUSTERED 
+   (
+	   [Name] ASC,
+	   [ReportUniqueName] ASC
+   )WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON, OPTIMIZE_FOR_SEQUENTIAL_KEY = OFF) ON [PRIMARY]
+   ) ON [PRIMARY]
+   GO
+   INSERT [dbo].[rn__Options] ([Name], [ReportUniqueName], [Value]) VALUES (N'PowerBIShowAllTables', '', N'1')
+   GO
+```
+
+The value of 1 for PowerBIShowAllTables will cause all snapshots to be shown in the navigation interface. Any other value will retain the default behavior.
+
+For Oracle, you may run the following script (this assumes that the mobie DataBox schema is named "MOBIE"):
+
+```
+  CREATE TABLE "MOBIE"."RN__OPTIONS" 
+   (	"NAME" VARCHAR2(32 BYTE), 
+	"REPORTUNIQUENAME" VARCHAR2(275 BYTE), 
+	"VALUE" VARCHAR2(256 BYTE)
+   )
+
+  CREATE UNIQUE INDEX "MOBIE"."RN__OPTIONS_PK" ON "MOBIE"."RN__OPTIONS" ("NAME", "REPORTUNIQUENAME") 
+
+  ALTER TABLE "MOBIE"."RN__OPTIONS" ADD CONSTRAINT "RN__OPTIONS_PK" PRIMARY KEY ("NAME", "REPORTUNIQUENAME")
+  ALTER TABLE "MOBIE"."RN__OPTIONS" MODIFY ("VALUE" NOT NULL ENABLE);
+  ALTER TABLE "MOBIE"."RN__OPTIONS" MODIFY ("REPORTUNIQUENAME" NOT NULL ENABLE);
+  ALTER TABLE "MOBIE"."RN__OPTIONS" MODIFY ("NAME" NOT NULL ENABLE);
+
+  INSERT INTO "MOBIE"."RN__OPTIONS" (NAME, REPORTUNIQUENAME, VALUE) VALUES ('PowerBIShowAllTables', ' ', '1')
+```
+
+Note that if you create user based permissions to your mobie database/schema, connecting users will also require SELECT permission to the rn__Options table.
```

Comment:
This would make more sense after the intro (prior to the instructions section.

Reply:
I can see moving it to the top of this instruction section, but there is no need to add the permissions if you aren't creating the table for this purpose, so I don't see that it makes sense to place elsewhere.